tests/ludwig/features/test_feature_utils.py

```diff
@@ -35,3 +37,20 @@ def test_ludwig_feature_dict_with_periods():
     assert feature_dict.keys() == ["to."]
     assert feature_dict.items() == [("to.", to_module)]
     assert feature_dict["to."] == to_module
+
+
+@pytest.mark.parametrize("sequence_type", [list, tuple, np.ndarray])
+def test_compute_sequence_probability(sequence_type):
+    inputs = [
+        [0.1, 0.2, 0.7],
+        [0.3, 0.4, 0.3],
+        [0.6, 0.3, 0.2],
+    ]
+    if sequence_type == tuple:
+        inputs = tuple(inputs)
+    elif sequence_type == np.ndarray:
```

Comment:
nit: Avoid logic in tests:
https://docs.microsoft.com/en-us/dotnet/core/testing/unit-testing-best-practices#avoid-logic-in-tests

Since there's only 3 types of input, I would prefer removing the parameterization and list out the different possible inputs explicitly, or change `np.ndarray` to `np.array` and then do

```
inputs = sequence_type([
        [0.1, 0.2, 0.7],
        [0.3, 0.4, 0.3],
        [0.6, 0.3, 0.2],
    ])

...
```

Reply:
Got it, good tip! I made the latter change.